lib/persisted-model.js

```diff
@@ -724,7 +724,8 @@ module.exports = function(registry) {
       description: 'Check whether a model instance exists in the data source.',
       accessType: 'READ',
       accepts: [
-        {arg: 'id', type: 'any', description: 'Model id', required: true},
+        {arg: 'id', type: 'any', description: 'Model id', required: true,
+          http: {source: 'path'}},
```

Comment:
When building Swagger Spec describing LoopBack REST API, we are always converting arguments that are found in the URL path into `http: {source: 'path'}}` - see [lib/specgen/route-helper.js#L296-L29](https://github.com/strongloop/loopback-swagger/blob/fe5cfd7a11fb4cb0c9804db510d461a3d2ad53a6/lib/specgen/route-helper.js#L296-L298).

On strong-remoting's side, when extracting remote method arguments from the incoming requests:
 - If the http source is 'path', the value is read from `req.params`, see [HttpContext.prototype.buildArgs](https://github.com/strongloop/strong-remoting/blob/dde88ef2d05f6e60ca8ffa3295b35926b6424481/lib/http-context.js#L158-L162)
- If the argument does not have any http source set, we try to read it from `req.params` first, which is the same behavior. See [HttpContext.prototype.getArgByName](https://github.com/strongloop/strong-remoting/blob/dde88ef2d05f6e60ca8ffa3295b35926b6424481/lib/http-context.js#L226-L236)

My conclusion is that the proposed change is safe and should not break any existing applications 👍